features/add_to_cart.feature

```diff
@@ -0,0 +1,17 @@
+Feature: As a user,
+  In order to choose multiple photos,
+  I need to be able to add them to the shopping cart
+
+Background:
+  Given The following photos exist
+    | title   | price   | image_path                              |
+    | Sporty1 | 30      | http://lorempixel.com/400/200/sports/1/ |
+    | Sporty2 | 40      | http://lorempixel.com/400/200/sports/2/ |
+    | Sporty3 | 50      | http://lorempixel.com/400/200/sports/3/ |
+    | Sporty4 | 60      | http://lorempixel.com/400/200/sports/4/ |
+
+Scenario: Add photo to cart
```

Comment:
There is no other user interaction except to press the add to cart on the item so we don't have any path there the user can make incorrect input or anything similar to that.